contracts/Conference.sol

```diff
@@ -103,6 +114,36 @@ contract Conference is Destructible, GroupAdmin {
 		participants[msg.sender] = Participant(_participant, msg.sender, false, false);
 	}
 
+	function unregisterInternal() internal {
+		// require(msg.value == deposit);
+		// require(isRegistered(msg.sender));
```

Comment:
Need to fix these -- should only unregister if sender is registered and within a certain time period (i.e. 1 day)